app/views/layouts/application.html.erb

```diff
@@ -12,32 +12,31 @@
   <body>
     <nav>
       <%= image_tag "H2L.png", id:"h2l" %>
+      <% if !session[:user_id].nil? %>
       <div class="nav-logged-in">
-        <%  %>
-        <!-- if !session[:user_id].nil? -->
-          <!-- Logged in as #{current_user.first_name} #{current_user.last_name} -->
-        <!-- end -->
-        <%  %>
+          <%= "Logged in as #{current_user.first_name} #{current_user.last_name}" %>
       </div>
+
       <div class="nav-dashboard">
-        <%= link_to "Dashboard", welcome_path, class:"nav-button" %>
-        <!-- link_to "Dashboard", dashboard_path, class:"nav-button" -->
+        <%= link_to "Dashboard", dashboard_path, class:"nav-button" %>
       </div>
-      <div class="nav-logout">
-        <%  %>
-        <!-- if !session[:user_id].nil? -->
-          <!-- link_to "Log Out", logout_path, class:"nav-button" -->
-        <!-- end -->
-        <%  %>
+
+      <div class="nav-account">
+        <%= link_to "My Account", account_path, class:'nav-button' %>
       </div>
-      <% if !flash.empty? %>
-      <div class="flash-message">
-        <% flash.each do |name, msg| %>
-        <%= content_tag :div, msg, class: name %>
-        <% end %>
+      
+      <div class="nav-logout">
+          <%= link_to "Log Out", logout_path, class:"nav-button" %>
       </div>
       <% end %>
     </nav>
+    <% if !flash.empty? %>
```

Comment:
Is `if !<statement>` preferable to `unless <statement>`?

Both seem pretty mind-bendy and it's probably mostly a question of style.

For this line in particular, I don't believe that it is necessary to check to see if `flash.empty?`. If it is empty, nothing, the below code simply will not display anything.

Reply:
Because the flash is styled, unless I put this if statement, a little empty box appears in the corner.